src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -646,6 +661,15 @@ public void run() {
 
         GT_Values.RA.addBlastRecipe(Materials.Zincite.getDust(2), Materials.Carbon.getDust(1), GT_Values.NF, Materials.CarbonDioxide.getGas(1000L), Materials.Zinc.getIngots(3), Materials.Ash.getDustTiny(2), 240, 120, 1200);
 
+        GT_Values.RA.addBlastRecipe(Materials.Silicon.getDust(1), GT_Utility.getIntegratedCircuit(11), Materials.Oxygen.getGas(2000), GT_Values.NF, Materials.Silicon.getIngots(1),  GT_Values.NI, 1600, 120, 1600);
+
+        GT_Values.RA.addBlastRecipe(Materials.Sapphire.getDust(3), GT_Utility.getIntegratedCircuit(1), GT_Values.NF, GT_Values.NF, Materials.Aluminiumoxide.getDust(1),  GT_Values.NI, 1200, 120, 1200);
```

Comment:
These new gem to alum recipes probably need a different circuit to prevent conflict with old gem to alum recipes.

Reply:
Gem to Aluminium ? I removed the old recipes. Was it possible to smelt sapphire to Alu ? I mean in gem form ?

Reply:
Line 571-575 in this file.

Reply:
and fixed to circuit 2